prometheus_speedtest/prometheus_speedtest.py

```diff
@@ -55,14 +54,14 @@ def test(self):
 
 class SpeedtestCollector():
     """Performs Speedtests when requested from Prometheus."""
-    def __init__(self, tester=None, server=0):
+    def __init__(self, tester=None, servers=[]):
```

Comment:
Default this to `None`. If you look at `get_best_servers` the `servers` argument defaults to None.

https://github.com/sivel/speedtest-cli/blob/c58ad3367bf27f4b4a4d5b1bca29ebd574731c5d/speedtest.py#L1427